lms/static/scripts/frontend_apps/services/grading.js

```diff
@@ -10,7 +10,7 @@ import { apiCall } from '../utils/api';
  * The fetched grade success result.
  *
  * @typedef FetchGradeResult
- * @prop {number} currentScore - The fetched grade
+ * @prop {number|null} [currentScore] - The fetched grade
```

Comment:
This type was incomplete. I have verified the possibility of `null` value in Blackboard.

Regarding adding also `undefined`, I did it as a defensive programming design just in case we encounter the situation the under unforeseen circumstances.